### PR TITLE
Special contributor mentions on the about screen

### PR DESCRIPTION
**Pull request by SerhiiGadzhilov**
SYNERGY1-1570

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -9,6 +9,7 @@ Enhancements:
 - #7183 Refactoring for Synergy 3 licenses
 - #7190 Maintenance license serial key support
 - #7193 Support for maintenance licenses in China
+- #7197 New design for the about screen
```
[the remark this answers is not in the document]
Comment (SerhiiGadzhilov) [reply ChangeLog]: Fixed

File: src/gui/src/CreditsLoader.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+CreditsLoader::CreditsLoader(MainWindow& mainWindow) :
+    m_mainWindow(mainWindow)
+{
+    connect(&m_manager, SIGNAL(finished(QNetworkReply*)), this, SLOT(replyFinished(QNetworkReply*)));
+}
+
+void CreditsLoader::loadEliteBackers()
+{
+    const auto ELIT_BACKERS_URL = "https://api.symless.com/credits/elite-backers";
```
[the remark this answers is not in the document]
Comment (SerhiiGadzhilov) [reply src/gui/src/CreditsLoader.cpp]: Fixed.
Thank you!

**Review by nbolton — DISMISSED** [review]

File: src/gui/src/AboutDialogBase.ui [hunk 3]
```diff
@@ -41,56 +41,202 @@
    <bool>true</bool>
   </property>
   <layout class="QGridLayout">
-   <item row="5" column="1" colspan="2">
-    <layout class="QHBoxLayout" name="horizontalLayout">
+   <item row="10" column="1">
+    <widget class="QLabel" name="labelCreditsLink">
+     <property name="text">
+      <string notr="true">&lt;a href=&quot;https://symless.com/synergy/credits&quot; style=&quot;color: #4285F4;&quot;&gt;View the complete credits&lt;/a&gt;</string>
```
Comment (nbolton) [inline src/gui/src/AboutDialogBase.ui:47]: @SerhiiGadzhilov

File: src/gui/src/AboutDialogBase.ui [hunk 4]
```diff
@@ -41,56 +41,202 @@
...
+     </property>
+     <property name="openExternalLinks">
+      <bool>true</bool>
+     </property>
+    </widget>
+   </item>
+   <item row="9" column="1">
+    <widget class="QLabel" name="labelEliteBackerLink">
+     <property name="text">
+      <string>&lt;a href=&quot;https://symless.com/synergy/account-status&quot; style=&quot;color: #4285F4;&quot;&gt;Become an Elite Backer&lt;/a&gt;</string>
```
Comment (nbolton) [inline src/gui/src/AboutDialogBase.ui:57]: @SerhiiGadzhilov

**Review by nbolton — CHANGES_REQUESTED** [review]

**Review by nbolton — APPROVED** [review]

File: src/gui/src/AboutDialogBase.ui (hunk 4 above)
Comment (SerhiiGadzhilov) [reply src/gui/src/AboutDialogBase.ui]: Fixed

**Pull request merged**